test/manifests/normal/scaleup.yaml

```diff
@@ -1,3 +1,4 @@
+location: "{{ .Env.AZURE_REGION }}"
```

Comment:
Did you find that this is needed?

Reply:
Discussed in bluejeans, seems to be required by the real RP? If so, open an issue to bring the fake RP inline.